### PR TITLE
Update DG archive and load

### LINKED ISSUE #110: Update UG to explain the archive and load

### PR DESCRIPTION
**Pull request by Jianwen0451**
close #110

### CODE REVIEW

**Comment by Jianwen0451** [comment]
The name of pull request and branch should saying update DG, please ignore this slight problem

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -261,10 +261,46 @@ _{more aspects and alternatives to be added}_
 
 ### \[Proposed\] Data archiving
 
-_{Explain here how the data archiving feature will be implemented}_
+The archive and load feature is achieved through `ArchiveCommand` and `LoadCommand` which both extend the `Command` class. When such command is executed, the LogicManager will update the Storage when necessary.
```
Comment (tanyqcollin) [inline docs/DeveloperGuide.md:264]: I think the first sentence should be `The archive and load features are ...`

**Review by tanyqcollin — APPROVED** [review]
a minor grammar mistake

**Review by RadieonAjax — APPROVED** [review]
LGTM!

**Pull request merged**